docs/getting_started/index.rst

```diff
@@ -466,18 +465,17 @@ Yielding the output:
 
 .. literalinclude:: xnvme_win_io_async_read_iocp_th.out
    :language: bash
-   :lines: 1-12
 
 Async I/O via ``io_ring``
-~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**xNVMe** utilizes the Windows **io_ring** interface, its support for
+**xNVMe** utilizes the Windows **ioring** interface, its support for
 feature-probing the **io_ring** interface and the **io_ring** opcodes:
 
 When available, then **xNVMe** can send the **ioring** specific request using
-**IORING_HANDLE_REF** and **IORING_BUFFER_REF** structure for **read** via
-the Windows **io_ring** interface. Doing so improves command-throughput at
-all io-depths when compared to sending the command via NVMe Driver IOCTLs.
+**IORING_HANDLE_REF** and **IORING_BUFFER_REF** structure for **read** via the
+Windows **io_ring** interface. Doing so improves command-throughput at all
```

Comment:
ioring instead of io_ring

Reply:
Yeah, unfortunately then currently then the backend-implementation is named ``io_ring``. This should probably be changed in the future, to align with the Windows Naming of ``IORING``.